spec/tapioca/cli_spec.rb

```diff
@@ -608,6 +665,32 @@ def body=(body); end
         end
       end
 
+      describe("with excluded files") do
+        before do
+          execute("dsl")
+        end
+
+        it "advises of removed file(s) and returns exit_status 1" do
+          output = execute("dsl", "--verify", exclude_generators: "SidekiqWorker")
```

Comment:
Can we add a test showing the behaviour for `exclude_generators: "NotFound"` please?

Reply:
Added, see last commit (`Error if requested/excluded generator does not exist`).